dev/how-to/maintain: update Usage Examples in slow query (#1190)
pingcap/docs: dev/how-to/maintain/identify-slow-queries.md

The slow log format is updated in TiDB v2.1.8 and later. For the slow query log information in versions earlier than v2.1.8, see [this file](https://github.com/pingcap/docs/blob/master/v2.1/sql/slow-query.md).

> **Note:**
>

> > There are very few field information of slow query log in TiDB 2.1.8 version. The field information mentioned in the following part is added later in TiDB 3.0.0 version.

Reviewer: ```suggestion
> There is very few field information of slow query logs in TiDB v2.1.8. Some of the field information mentioned in the following part is added in TiDB v3.0.0.
```

* `Txn_start_ts`: The start timestamp of the transaction (transaction ID). You can use this value to `grep` the transaction-related logs.
* `User`: The name of the user who executes this statement.
* `Conn_ID`: The Connection ID (session ID). For example, you can use the keyword `con:3` to `grep` the log whose session ID is 3.
* `DB`: The current database.
* `Index_ids`: The IDs of the indexes involved in the statement.

> * `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, such as `analyze`, `load variables`, etc. `false` indicates the SQL statement is executed by the user.

Reviewer: ```suggestion
* `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, such as `analyze`, `load variables`, etc. `false` indicates that the SQL statement is executed by the user.
```

Reviewer: Your expression is right. But in technical writing, it is recommended to add "that" here.

* `Digest`: The fingerprint of the SQL statement.

> * `Memory_max`: Indicates the maximum memory space used during the execution period of this SQL statement (the unit is byte).

Reviewer: ```suggestion
* `Memory_max`: The maximum memory space used during the execution period of this SQL statement (the unit is byte).
```

> * `Query_time`: 16.479155653

Reviewer: Why did you update this line?

Author: Sorry, I confused it with that in the example above.

|            |   `Time` timestamp unsigned NULL DEFAULT NULL,              |
|            |   `Txn_start_ts` bigint(20) unsigned DEFAULT NULL,          |
|            |   `User` varchar(64) DEFAULT NULL,                          |
|            |   `Conn_ID` bigint(20) unsigned DEFAULT NULL,               |
|            |   `Query_time` double unsigned DEFAULT NULL,                |
|            |   `Process_time` double unsigned DEFAULT NULL,              |
|            |   `Wait_time` double unsigned DEFAULT NULL,                 |
|            |   `Backoff_time` double unsigned DEFAULT NULL,              |
|            |   `Request_count` bigint(20) unsigned DEFAULT NULL,         |
|            |   `Total_keys` bigint(20) unsigned DEFAULT NULL,            |
|            |   `Process_keys` bigint(20) unsigned DEFAULT NULL,          |
|            |   `DB` varchar(64) DEFAULT NULL,                            |
|            |   `Index_ids` varchar(100) DEFAULT NULL,                    |
|            |   `Is_internal` tinyint(1) unsigned DEFAULT NULL,           |
|            |   `Digest` varchar(64) DEFAULT NULL,                        |

> |            |   `Query` varchar(4096) DEFAULT NULL                        |

Reviewer: You missed Line 92 - Line 101 in https://github.com/pingcap/docs-cn/pull/1278/files.

Author: Addressed, thanks for reminding me about it.
